packages/request-manager/tests/interceptor-whitelist.test.ts

```diff
@@ -6,8 +6,12 @@ import WebSocket from 'ws';
 import { createInterceptor } from '../src';
 import { InterceptorOptions } from '../src/types';
 
-const WHITELISTED_DOMAIN = 'tbtc1.trezor.io';
-const NOT_WHITELISTED_DOMAIN = 'tbtc2.trezor.io';
+const WHITELISTED_DOMAIN = 'url-that-is-valid-and-white-listed-but-does-not-exists.com';
+const NOT_WHITELISTED_DOMAIN = 'not-white-listed-but-does-not-exists.com';
+
+const OK_ERROR_HTTPS = `request to https://${WHITELISTED_DOMAIN}/ failed, reason: getaddrinfo ENOTFOUND ${WHITELISTED_DOMAIN}`;
+
+const OK_ERROR_SHORT = `getaddrinfo ENOTFOUND ${WHITELISTED_DOMAIN}`;
```

Comment:
Nit: expect.toThrow accepts regex, so you can generalize it, so it doesn't rely on exact wording of the error.
```suggestion
const OK_ERROR_SHORT = `getaddrinfo ENOTFOUND ${WHITELISTED_DOMAIN}`;
const OK_ERROR_HTTPS = new RegExp(`.*${OK_ERROR_SHORT}`);
```